templates/macros/_macros-quote.jinja

```diff
@@ -129,21 +129,23 @@
     {% endif -%}
     {#- Skip to fourth column if the signpost_image (which takes up first three columns) is missing -#}
     <div class="col-9{% if not has_signpost_image %} col-start-large-4{% endif %}">
-      <hr class="u-hide--medium u-hide--small">
-      {% if has_citation -%}
-        <!-- When a citation is present, wrap the quote and citation in a nested grid to space them properly -->
-        <div class="row">
-          <div class="col-6">
-            {{ _quote_body() }}
-          </div>
-          <div class="col-3">
-            {{ _citation_block() }}
+      <hr class="u-hide--medium u-hide--small is-muted">
+      <div class="p-section--shallow">
```

Comment:
@jmuzina I added a `p-section--shallow` wrapper to the quote_body() per Lyubo's design comment. Lmk if this is okay, otherwise I will create a custom case study quote macro that adds the wrapper to this existing macro.

Reply:
Hey @britneywwc , I'm checking with Lyubo to see if this is the correct place to put this shallow section, I'll get back to you (and hopefully approve this PR finally) tomorrow.

Reply:
Thanks! :)